easybuild/easyconfigs/c/CalculiX/calculix-2.7.make.patch

```diff
@@ -0,0 +1,30 @@
+--- CalculiX/ccx_2.7/src/Makefile.orig	2014-10-15 14:46:37.755959000 +1300
++++ CalculiX/ccx_2.7/src/Makefile	2014-10-16 09:29:07.357968000 +1300
+@@ -1,9 +1,9 @@
+ 
+-CFLAGS = -Wall -O3  -I ../../../SPOOLES.2.2 -DARCH="Linux" -DSPOOLES -DARPACK -DMATRIXSTORAGE
++CFLAGS = -Wall -O3  -I $(EBROOTSPOOLES) -DARCH="Linux" -DSPOOLES -DARPACK -DMATRIXSTORAGE
+ FFLAGS = -Wall -O3
+ 
+ CC=cc
```

Comment:
ah, then change it to "CC:=$(CC)"

see the "simply expanded variables" part @ https://www.gnu.org/software/make/manual/make.html#Flavors